httpapi/coreapi/service.go

```diff
@@ -0,0 +1,23 @@
+package coreapi
```

Comment:
This package will be all document agnostic, correct?

Reply:
yeap